task1-simple-program/get_random_number.c

```diff
@@ -0,0 +1,14 @@
+#include <stdlib.h>
+#include <time.h>
+#include "get_random_number.h"
+
+static unsigned char randomized;
```

Comment:
Because randomized is used only in get_random_number(), please move it inside the method; ototherwise you allow modifying this variable from any method of the file.